addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -650,84 +725,117 @@ protected boolean isInMotion() {
     public void setChargeLimit(int percent) {
         JsonObject payloadObject = new JsonObject();
         payloadObject.addProperty("percent", percent);
-        sendCommand(TESLA_COMMAND_SET_CHARGE_LIMIT, gson.toJson(payloadObject), commandTarget);
-        requestData(TESLA_CHARGE_STATE);
+        sendCommand(COMMAND_SET_CHARGE_LIMIT, gson.toJson(payloadObject), commandTarget);
+        requestData(CHARGE_STATE);
     }
 
     public void setSunroof(String state) {
         JsonObject payloadObject = new JsonObject();
         payloadObject.addProperty("state", state);
-        sendCommand(TESLA_COMMAND_SUN_ROOF, gson.toJson(payloadObject), commandTarget);
-        requestData(TESLA_VEHICLE_STATE);
+        sendCommand(COMMAND_SUN_ROOF, gson.toJson(payloadObject), commandTarget);
+        requestData(VEHICLE_STATE);
     }
 
     public void moveSunroof(int percent) {
         JsonObject payloadObject = new JsonObject();
         payloadObject.addProperty("state", "move");
         payloadObject.addProperty("percent", percent);
-        sendCommand(TESLA_COMMAND_SUN_ROOF, gson.toJson(payloadObject), commandTarget);
-        requestData(TESLA_VEHICLE_STATE);
+        sendCommand(COMMAND_SUN_ROOF, gson.toJson(payloadObject), commandTarget);
+        requestData(VEHICLE_STATE);
     }
 
     public void setTemperature(float temperature) {
         JsonObject payloadObject = new JsonObject();
         payloadObject.addProperty("driver_temp", temperature);
         payloadObject.addProperty("passenger_temp", temperature);
-        sendCommand(TESLA_COMMAND_SET_TEMP, gson.toJson(payloadObject), commandTarget);
-        requestData(TESLA_CLIMATE_STATE);
+        sendCommand(COMMAND_SET_TEMP, gson.toJson(payloadObject), commandTarget);
+        requestData(CLIMATE_STATE);
+    }
+
+    public void openFrunk() {
+        JsonObject payloadObject = new JsonObject();
+        payloadObject.addProperty("which_trunk", "front");
+        sendCommand(COMMAND_TRUNK_OPEN, gson.toJson(payloadObject), commandTarget);
+        requestData(VEHICLE_STATE);
+    }
+
+    public void openTrunk() {
```

Comment:
Using `openRearTrunk() ` here would make it more obvious which trunk this is about.